scripts/run_nodes.sh

```diff
@@ -96,8 +96,11 @@ run_node() {
     --execution Native \
     --rpc-cors=all \
     --no-mdns \
+    --public-validator-addresses 127.0.0.1:$((30343 + i)) \
+    --validator-port $((30343 + i)) \
```

Comment:
based on that it looks like the ort of `public-validator-addresses` should match with `validator-port`. Consider removing port part from the `public-validator-addresses` and get full address by combining it with `validator-port`